packages/virtualized-lists/Lists/VirtualizedSectionList.js

```diff
@@ -593,14 +593,21 @@ function ItemWithSeparator(props: ItemWithSeparatorProps): React.Node {
       {...separatorProps}
     />
   );
-  return leadingSeparator || separator ? (
+  const renderSeparator = leadingSeparator || separator;
+  return (
     <View>
-      {inverted === false ? leadingSeparator : separator}
+      {renderSeparator
+        ? inverted === false
+          ? leadingSeparator
+          : separator
+        : null}
       {element}
-      {inverted === false ? separator : leadingSeparator}
+      {renderSeparator
+        ? inverted === false
+          ? separator
+          : leadingSeparator
+        : null}
     </View>
```

Comment:
Instead of a View could this just return a Fragment? Is the view essential for layout reasons?

Reply:
yes it is necessary @javache.

Reply:
@javache Thanks for the feedback on this one, let me know what we can do about this to land this :-).

Reply:
What breaks in terms of layout when you remove the View? They carry some overhead, so if we can achieve the correct layout in a different way, that would be preferable.

Reply:
ok let me try if we can do the same.